app/components/ak-pagination-provider/index.ts

```diff
@@ -14,7 +16,7 @@ export type PaginationItemPerPageOptionProps = {
 };
 
 interface PaginationProviderDefaultBlockHash<R> {
-  currentPageResults: R[];
+  currentPageResults: R[] | DS.AdapterPopulatedRecordArray<R> | undefined;
```

Comment:
Can you explain some more? 

`DS.AdapterPopulatedRecordArray<R>` was added because it is a valid record format and was used in notifications. R represents the model and is inferred from the `results` value supplied to the pagination provider.